robot_controll/Config.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Config.h"
+
+YAML::Node Config::config = YAML::LoadFile("../config.yaml");
```

Comment:
That is true, but now i think it is not necessary - we will need to reload whole application after config change.